pkgs/data/fonts/rictydiminished-with-firacode/default.nix

```diff
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [
     fontforge
-    (pythonFull.withPackages (ps: [
+    (python3Full.withPackages (ps: [
```

Comment:
why is it using pythonFull? Please try without

Reply:
It seems python3 is sufficient.